src/cljs/zetawar/views.cljs

```diff
@@ -384,7 +384,12 @@
                                  :on-hide hide-settings}
      [:> js/ReactBootstrap.Modal.Header {:close-button true}
       [:> js/ReactBootstrap.Modal.Title
-       (translate :configure-faction-title)]]
+       (str "Configure Faction: "
+            (when @faction
```

Comment:
What's the reasoning behind having both conn and faction as arguments? Isn't only one necessary to get the desired result?
For example, with faction:
```
(deftrack faction-color-name [faction]
  (when @faction
    (-> @faction ...
```

Or with conn:
```
(deftrack faction-color-name [conn]
  (when-let [faction @(faction-to-configure conn)]
    (-> faction ...
```

Every other deftrack in subs uses conn, so the second would probably be more appropriate.

I might just be missing something obvious.